--- a/CMakeLists.txt	2019-07-03 08:07:57.467038400 -0500
+++ b/CMakeLists.txt	2019-07-03 08:05:19.801540398 -0500
@@ -538,7 +538,7 @@ if(PNG_SHARED)
   endif()
   target_link_libraries(png ${ZLIB_LIBRARY} ${M_LIBRARY})
 
-  if(UNIX AND AWK)
+  if(UNIX AND AWK AND NOT ANDROID)
     if(HAVE_LD_VERSION_SCRIPT)
       set_target_properties(png PROPERTIES LINK_FLAGS
         "-Wl,--version-script='${CMAKE_CURRENT_BINARY_DIR}/libpng.vers'")
